automation/check-patch.setup.sh

```diff
@@ -25,3 +25,7 @@ cp -rf $(pwd)/. ${TMP_PROJECT_PATH}
 
 echo 'Exporting temporary project path'
 export TMP_PROJECT_PATH
+
+echo 'Exporting KubeMacPool component project path'
+KUBEMACPOOL_TMP_PROJECT_PATH=${TMP_PROJECT_PATH}/_components/go/src/github.com/k8snetworkplumbingwg/kubemacpool/
```

Comment:
This should not be in the shared setup script as it is irrelevant for most of the targets.